simdpp/detail/insn/cmp_neq.h

```diff
@@ -192,7 +192,7 @@ mask_int64x2 i_cmp_neq(const uint64x2& a, const uint64x2& b)
     return bit_not(cmp_eq(a, b));
 #elif SIMDPP_USE_SSE2
     uint64x2 r32, r32s;
-    r32 = (uint32x4)cmp_eq(uint32x4(a), uint32x4(b));
```

Comment:
I don't understand why uint32x4 is used and not uint64x2?

Reply:
Me neither.

Reply:
Good, I'm not alone :-)